gradle.properties

```diff
@@ -17,23 +17,12 @@ mavenCentralPassword=
 
 # Configures whether release is an unstable snapshot.
 IS_SNAPSHOT=true
+RELEASE_SIGNING_ENABLED=false
 
-LIBRARY_VERSION=1.6.0-beta01
-POM_NAME=Zcash Android Wallet SDK
-GROUP=cash.z.ecc.android
-POM_ARTIFACT_ID=zcash-android-sdk
-POM_DESCRIPTION=This lightweight SDK connects Android to Zcash. It welds together Rust and Kotlin in a minimal way, allowing third-party Android apps to send and receive shielded transactions easily, securely and privately.
-POM_INCEPTION_YEAR=2018
-POM_URL=https://github.com/zcash/zcash-android-wallet-sdk/
-POM_SCM_URL=https://github.com/zcash/zcash-android-wallet-sdk/
-POM_SCM_CONNECTION=scm:git:git://github.com/zcash/zcash-android-wallet-sdk.git
-POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/zcash/zcash-android-wallet-sdk.git
-POM_LICENCE_NAME=The MIT License
-POM_LICENCE_URL=http://opensource.org/licenses/MIT
-POM_LICENCE_DIST=repo
-POM_DEVELOPER_ID=zcash
-POM_DEVELOPER_NAME=Zcash
-POM_DEVELOPER_URL=https://github.com/zcash/
+# Required by the maven publishing plugin
+SONATYPE_HOST=DEFAULT
+
+LIBRARY_VERSION=1.5.0-beta01
```

Comment:
Just a question on the lib version we release. Is this downgrading from 1.6.0-beta01 to 1.5.0-beta01 alright? As I can see we already released the 1.5.0 version earlier.